### DISCUSSION TITLE
Deleted torrents reappearing after container restart

### DISCUSSION THREAD

**Discussion by DrR0Ck** [haugene/docker-transmission-openvpn#2343] [General]
I am finding that when I attempt to clean up multiple torrents via deleting from the front end, that the deleted torrents reappear after a restart of the container.  This is problematic as I do not want the torrents downloaded again, and I then have to hunt down the .torrent files based on the hash value, which is kind of a pain.  So I am wondering whether this is a bug or if I might have something misconfigured.  Any assistance would be appreciated.  Thanks!

**Comment by pkishino** [comment]
There were some instances where this might happen when switching back to 2.94 from 3.0 transmission client.. need to stop, clean the torrent folder then restart

**Reply by DrR0Ck** [reply]
Thanks for your response.  When you say "clean the torrent folder," do you mean delete out all of the torrent files in there?  Or would it be something more selective?

**Comment by pkishino** [comment]
Yeah, delete the torrents and resume files.. as the two versions use
different hashing algorithms

**Reply by DrR0Ck** [reply]
I stopped the container, deleted the torrents and resumes, and when I restarted the container, I had no torrents.  I then stopped everything and moved the torrents back in, and restarted, and all the torrents were there, but I had lost all my stats and dates.  So I put the resumes back in and everything was back.  Was I supposed to delete the torrents/resumes while the container was running?  I assumed that if I was deleting the files that there was some master database that had the information, but I gather that is not the case - at least not across sessions.

**Reply by pkishino** [reply]
As you moved back transmission versions your best move is to get rid of all resumes and move the torrents out, then only re-add the ones you want to continue seeding and let transmission re-verify.

**Reply by DrR0Ck** [reply]
Got it.  Understood.  Will give it a go.

**Reply by DrR0Ck** [reply]
Is there any way to do this and not be starting from scratch as far as torrent dates, totals uploaded, etc.?

**Reply by DrR0Ck** [reply]
Or any way to export out this info and overlay it back in?

**Comment by pkishino** [comment]
Which info? Client stats are saved in stats.json file in transmission-home,
this isn’t affected by deleting the .resume and .torrent files.

**Reply by DrR0Ck** [reply]
The per-torrent stats.  Each torrent has date added info and the uploaded ever (total) column. these would be the ones I would most want to keep.  I was able to clean out all of the .torrent and .resume files for old stuff I am no longer seeding, and then I did re-checks on everything still seeding.  I guess this isn't going to get the job done, is it?

**Comment by pkishino** [comment]
Can I ask what you need that date for? Seems pointless to me.. but no,
that’s not going to work. Either you upgrade to 3.0 client (either switch
back to 3.7.1 or upgrade transmission on a startup script, check discussion
thread) or you remove all torrents and only re-add those you want to keep
and restart

**Reply by DrR0Ck** [reply]
I have a finite amount of storage to devote to torrents.  I use the dates combined with the upload totals to periodically decide which torrents to keep and which to delete to make room for new ones.

Regardless, I appreciate the help.  This is not a life and death type problem.  I'll delete everything and re-add the torrents I want to keep if that ensures it will make things work.  I prefer not to roll back to an older version and forego updates, and I prefer not to upgrade my instance to a Transmission version different from the one in the container releases as I will then have to upgrade every time a new version rolls out.
